packages/components/molecules/f-restaurant-card/src/components/RestaurantCard.vue

```diff
@@ -474,11 +514,28 @@ export default {
             order: initial;
         }
     }
+
+    .c-restaurantCard-ratingSkeleton {
+        .c-restaurantCard--listItem .c-restaurantCard-content & {
+            grid-column: 1;
+            width: 100%;
+        }
+    }
+
+    .c-restaurantCard--hideOnMob {
+        .c-restaurantCard--listItem .c-restaurantCard-content & {
+            display: block;
+        }
+    }
 }
 
 // forces elements after to a new line within a flex container/grid
 .c-restaurantCard-clearfix {
     width: 100%;
     margin: 0;
 }
+
+.c-restaurantCard--hideOnMob {
+    display: none;
+}
```

Comment:
I'm assuming this means _hide on mobile_, but I don't see a media query 🤔

Reply:
it's hidden on mobile by default, I wasn't sure what the best name should be but it's essentially only displayed on desktop when it's a list item. Which to simplify, is never show on mobile :D